packages/client/src/client.js

```diff
@@ -269,15 +270,17 @@ export class PercyClient {
   // created from `content` if one is not provided.
   async uploadResource(buildId, { url, sha, filepath, content } = {}) {
     validateId('build', buildId);
-    this.log.debug(`Uploading resource: ${url}...`);
     if (filepath) content = await fs.promises.readFile(filepath);
+    let encodedContent = base64encode(content);
+
+    this.log.debug(`Uploading ${formatBytes(encodedContent.length)} resource: ${url}...`);
```

Comment:
Can we also add a warning if this is over 20MB and an error text warning if its over 25M?

Reply:
added